### PR TITLE
Update common submodule

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**
Please make sure every Pull Request has the following information:

* A reference to a related issue in your repository (mandatory)
* A description of the changes proposed in the pull request
* @mentions of the person required for reviewing proposed changes (optional)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,12 @@
 buildscript {
     apply from: rootProject.file("gradle/versions.gradle")
+    println rootProject
```
Comment (AdamBJohnsonx) [inline build.gradle:3]:
```suggestion
```

File: build.gradle [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
     repositories {
         google()
         jcenter()
     }
-
+    println "com.android.tools.build:gradle:$rootProject.ext.gradleVersion"
```
Comment (AdamBJohnsonx) [inline build.gradle:9]:
```suggestion

```

File: build.gradle [hunk 3]
```diff
@@ -5,7 +5,6 @@ buildscript {
         google()
         jcenter()
     }
-
     dependencies {
```
Comment (AdamBJohnsonx) [inline build.gradle:8]:
```suggestion

    dependencies {
```

File: gradle/versions.gradle [hunk 4]
```diff
@@ -8,7 +8,7 @@ ext {
     buildToolsVersion = "28.0.3"
```
Comment (gitaumoses4) [inline gradle/versions.gradle:8]: I see the build tools were updated to `29.0.3` on `travis.yml`. How comes we are not doing the same here?

Comment (AdamBJohnsonx) [reply gradle/versions.gradle:8]: The build tools version is ignored, in any event.

Comment (gitaumoses4) [reply gradle/versions.gradle:8]: Ah. I see. Thank you for the clarification.

Comment (AdamBJohnsonx) [reply gradle/versions.gradle:8]: Eh - we should really clean up the Gradle files.  If this works, I may clean out some of these warnings before it merges.

**Comment by AdamBJohnsonx** [comment]
Think this Is now moot.

**Pull request closed without merging**